package.json

```diff
@@ -58,43 +59,46 @@
   },
   "private": true,
   "dependencies": {
-    "@angular-material-components/datetime-picker": "6.0.0",
-    "@angular-material-components/moment-adapter": "6.0.0",
-    "@angular/animations": "^11.2.14",
-    "@angular/cdk": "^11.2.13",
-    "@angular/common": "^11.2.14",
-    "@angular/compiler": "^11.2.14",
-    "@angular/core": "^11.2.14",
-    "@angular/forms": "^11.2.14",
-    "@angular/material": "^11.2.13",
-    "@angular/platform-browser": "^11.2.14",
-    "@angular/platform-browser-dynamic": "^11.2.14",
-    "@angular/router": "^11.2.14",
+    "@angular-material-components/datetime-picker": "15.0.0",
+    "@angular-material-components/moment-adapter": "15.0.0",
+    "@angular/animations": "^15.2.9",
+    "@angular/cdk": "^15.2.9",
+    "@angular/common": "^15.2.9",
+    "@angular/compiler": "^15.2.9",
+    "@angular/core": "^15.2.9",
+    "@angular/forms": "^15.2.9",
+    "@angular/material": "^15.2.9",
+    "@angular/platform-browser": "^15.2.9",
+    "@angular/platform-browser-dynamic": "^15.2.9",
+    "@angular/router": "^15.2.9",
+    "@circlon/angular-tree-component": "^11.0.4",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "5.0.14-angular11-upgrade",
-    "@hmcts/ccpay-web-component": "5.0.4",
+    "@hmcts/ccd-case-ui-toolkit": "7.0.0-NodeAndNgUpgrades.2-material-legacy",
+    "@hmcts/ccpay-web-component": "6.0.0-beta01",
```

Comment:
Not for immediate action, but just a reminder for before merge that we'll need F&P to provide a proper release

Reply:
Noted.  I am waiting for them to give final version